packages/sdk/src/libs/clients/lsp-creator/client.ts

```diff
@@ -0,0 +1,48 @@
+import assert from "assert";
+import { LongShortPairCreator__factory, LongShortPairCreator } from "@uma/core/contract-types/ethers";
+import Artifacts from "@uma/core/build/contracts/LongShortPairCreator.json";
```

Comment:
yea, my main concern is that we maintain ability to run on the frontend, from what i remember this was not compatible, though i could be wrong. on a higher level point though, it might be too hard to keep FE compatibility, so maybe its a pipe dream, what do you think about this

Reply:
That's fair! We should ultimately make these core files compatible with frontends, but I think it's fine to leave this as is until we get there. We just have to be mindful that these paths can break.